nxc/connection.py

```diff
@@ -554,7 +563,7 @@ def login(self):
                         return True
 
     def mark_pwned(self):
-        return highlight(f"({pwned_label})" if self.admin_privs else "")
+        return highlight(f" ({pwned_label})" if self.admin_privs else "")
```

Comment:
If you add the space here you probably have to change all other protocols, which use the function. Imo we should just leave it as it is, as the string where it is placed into should have the formatting anyways.

Reply:
Fair enough